examples/twittermap/web/public/javascripts/map/pointmap.js

```diff
@@ -158,66 +158,95 @@ angular.module('cloudberry.map')
         $scope.scale_x = 0;
         $scope.scale_y = 0;
 
-        //To generate Tweet Popup content from Twitter API (oembed.json?) response JSON
-        function translateOembedTweet(tweetJSON) {
-          var userName = "";
-          try {
-            userName = tweetJSON.author_name;
-          }
-          catch (e){
-            console.log("author_name missing in this Tweet.:" + e.message);
-          }
+        //shows each point's info in Front-end.
+        function translateTweetdatatoShow(tweetJSON) {
+            var tweetid = "";
+            try {
+                tweetid = tweetJSON["id"];
+            }
+            catch (e){
+                console.log("tweetid missing in this Tweet. :" + e.message);
+            }
```

Comment:
These console.log() are really necessary?

Reply:
Technically, we don't need this try catch and console.log, because now I could get every point's tweet. I keep this because Qiushi used to write these try-catch structure in his function, and i think it is a good habit to code.